test/test_parser.py

```diff
@@ -7,6 +7,7 @@
 
 from . import KaosTestCaseNonPersistent
 from .context import kaos
+import numpy
```

Comment:
Can you also do `import numpy as np` here? It's good to keep this consistent in case we ever need to move stuff around.